### PR TITLE
Fix some swedish strings

### PR DESCRIPTION
**Pull request by magol**

### CODE REVIEW

File: web/locales/sv.json [hunk 1]
```diff
@@ -328,8 +328,8 @@
         "NP": "Nepal",
         "NR": "Nauru",
         "NU": "Niue",
-        "NZ-NZA": "New Zealand (Aucklandöarna)",
-        "NZ-NZC": "New Zealand (Chathamöarna)",
+        "NZ-NZA": "Nya Zeeland (Aucklandöarna)",
```
Comment (corradio) [inline web/locales/sv.json]: Those two zones do not exist. Can you remove them?

**Comment by corradio** [comment]
@magol did you have time to view my comment?

**Comment by magol** [comment]
@corradio  Sorry for the delay
But the NZ-NZA and NZ-NZC do exists on the map and in en.json, https://github.com/tmrowco/electricitymap/blob/master/web/locales/en.json#L901

Should I remove the islands in en.json to?

**Comment by corradio** [comment]
My bad! They do exist!

**Pull request merged**